testing/README.md

```diff
@@ -40,21 +41,23 @@ If you feel that mocha/ sinon/ chai do not do a good job of addressing the way y
 
 - [testdouble](https://github.com/testdouble/testdouble.js) is a great alternative to sinon as a test double library.
 
+If your project is using [ReactJS](https://reactjs.org/) or is built with [Sewing-kit](https://github.com/Shopify/sewing-kit), we recommend using the following tools:
+- [Jest](http://facebook.github.io/jest/) for the test framework. 
+- [Enzyme](http://airbnb.io/enzyme/) for handling spies and mocks.
+
 ### Test Runners
 
 In addition to the testing framework, you generally need a test *runner*. Which runner to use will depend on the nature of your project:
 
-- For pure-JavaScript projects that do not need access to the DOM, avoid using a test runner and simply run the CLI for your test framework instead (all of the projects listed above have some form of CLI). If you need a simple DOM without actual rendering, you can also use [JSDom](https://github.com/tmpvar/jsdom) to construct a fake DOM.
-
-- For Non-Rails projects that require a full DOM, we recommend [Karma](https://karma-runner.github.io/1.0/index.html). It integrates well with other tools you are likely using, such as bundlers and code coverage tools. Feel free to instead use [Testem](https://github.com/testem/testem), another great JavaScript test runner, if you would like to try something a little different.
+- For projects using [Jest](http://facebook.github.io/jest/), you do not need an additional test runner. Just use Jest to run your tests and generate test coverage. See [Coverage](#coverage) for more information on test coverage with Jest.  
```

Comment:
Good to call out here: Jest will default to providing you a "fake" browser environment with JSDom. This is enough in most cases, but if you want to test things in a true browser environment it will not be sufficient to rely on Jest alone (unsure if Jest can be run in the browser yet, it couldn't in the past).